airflow-core/src/airflow/ui/public/i18n/locales/en/common.json

```
@@ -102,6 +102,7 @@
     "dags": "Dags",
     "docs": "Docs",
     "home": "Home",
+    "legacyFabViews": "Legacy FAB Views",
```

Text is a bit tight in the menu, can you change it to "Legacy Views"?
```suggestion
    "legacyFabViews": "Legacy Views",
```

---

See also screenshow above...

---

Yes, thanks! Done

---

Thanks for doing that. I was thinking that "FAB" isn't necessary for regular users to know